client/src/partials/cash/cash.html

```diff
@@ -13,29 +13,43 @@
     <div class="row">
 
       <!-- Switch Cashbox Form and Transfer -->
-      <div class="col-md-4 col-md-push-8 pad-bottom">
-        <p>
-          <button class="btn btn-warning" ng-click="CashCtrl.changeCashbox()" data-change-cashbox>
-            <span class="glyphicon glyphicon-inbox"></span> {{ "CASH.VOUCHER.CASHBOXES.CHANGE_CASHBOX" | translate }}
-          </button>
-        </p>
-
-        <p>
-          <button class="btn btn-primary" ng-click="CashCtrl.openTransferModal()" data-perform-transfer>
-            <span class="glyphicon glyphicon-export"></span> {{ "CASH.VOUCHER.CASHBOXES.TRANSFER" | translate }}
-          </button>
-        </p>
+      <div class="col-md-2">
+
+        <button class="btn btn-warning btn-block" ng-click="CashCtrl.openSelectCashboxModal()" data-change-cashbox>
+          <span class="glyphicon glyphicon-inbox"></span> {{ "CASH.VOUCHER.CASHBOXES.CHANGE_CASHBOX" | translate }}
+        </button>
+
+        <button class="btn btn-primary btn-block" ng-click="CashCtrl.openTransferModal()" data-perform-transfer>
+          <span class="glyphicon glyphicon-export"></span> {{ "CASH.VOUCHER.CASHBOXES.TRANSFER" | translate }}
+        </button>
+
+        <div class="alert alert-info btn-block">
+          <p style="margin-bottom: 10px;" ng-if="!CashCtrl.cashbox.id">
+            <i class="glyphicon glyphicon-info-sign"></i> {{ "CASH.VOUCHER.CASHBOXES.UNCONFIGURED" | translate }}
```

Comment:
For future reference, `<span>`s are preferred for icons rather than `<i>`s in the bootstrap documentation.  Since I don't know which is better, let use `<span>`s.